assembl/static2/js/app/pages/home.jsx

```diff
@@ -7,7 +7,7 @@ import Video from '../components/home/video';
 import Twitter from '../components/home/twitter';
 import Chatbot from '../components/home/chatbot';
 import Partners from '../components/home/partners';
-import ScrollOnePageButton from '../components/common/scrollOnePageButton';
+import ScrollOnePageButtonWithScreenHeight from '../components/common/scrollOnePageButton';
```

Comment:
I prefer to keep the component's name in sync with the filename (so that when you are looking to react tab in your browser, you know which file you have to look at for the component you are interested in). The solution to the eslint error here is probably to rename `ScrollOnePageButton` component (but to what ? `ScrollOnePageButtonWithoutScreenHeight` ? arg, ugly ;-) )

Reply:
Ugly but better to have to use that in tests and keep the clean name in the actual code, will change